pkg/continuoustest/client.go

```diff
@@ -254,6 +262,10 @@ func (rt *clientRoundTripper) RoundTrip(req *http.Request) (*http.Response, erro
 		req.Header.Set("Cache-Control", "no-store")
 	}
 
-	req.Header.Set("X-Scope-OrgID", rt.tenantID)
+	if rt.basicAuthUser != "" && rt.basicAuthPassword != "" {
```

Comment:
Could we add some validation above (with the other configuration validation) to make sure basic auth and tenant ID aren't both set?